### ISSUE TITLE
Show 24h and 7d APR on the swap pools UI

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/portal#590]
## Context
Currently there is only 7d APR shown on the swap pools screen, this ticket is to show both 24h and 7d APR

## Todo
* Change the text "APR" to "APR(24h/7d)"
* Below for each asset show both values i.e 24h and 7d APR 
* Attached reference of Nabla UI

## Note
<img width="951" alt="Screenshot 2024-10-04 at 5 31 31 PM" src="https://github.com/user-attachments/assets/424fc090-2783-408c-98db-060ef8d0ff33">

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/590?planning-poker) with Zenhub @ebma @Sharqiewicz @TorstenStueber

**Comment by ebma** [comment]
The indexer currently only exposes a general `apr` field. If we want to split this into `24h` and `7d`, we'd need to touch the indexer first.

**Comment by TorstenStueber** [comment]
@prayagd What is the definition of a 24 hour and a 7 day APR? It's an oxymoron, isn't it as APR is an annual rate?

Anyway, whatever that means. We calculate the APR extrapolated from the gains within the last seven days.

**Comment by prayagd** [comment]
> We calculate the APR extrapolated from the gains within the last seven days.

What we are extrapolating using APR for 7 days we want to do the same for 24h

**Comment by prayagd** [comment]
> What we are extrapolating using APR for 7 days we want to do the same for 24h

Even for doing this we need to make changes to the indexer?

**Comment by ebma** [comment]
Well the apr field is currently exposed by the indexer. And the nabla team also gets it from their indexer, so I think it makes most sense to do this. I found the relevant lines of code that describe the calculation they do for the 24h and 7d apr [here](https://github.com/NablaFinance/subgraph-mainnetAlpha/blob/20f9a57f8e376aa4fdd1062ea0900bc25edc4e4a/src/lib/pool-updates.ts#L118-L122).

**Comment by TorstenStueber** [comment]
Okay, understood. But is this high priority for us? I would consider it more a nice to have – we don't need to copy every aspects of the Nabla UI at this stage of the project.

**Comment by prayagd** [comment]
> Okay, understood. But is this high priority for us? 
Not a high priority, good to have in the backlog

**Comment by ebma** [comment]
@pendulum-chain/product as I pointed out, this ticket requires changes to the indexer before it can be worked on. I created the ticket https://github.com/pendulum-chain/pendulum-squids/issues/74 for you, please prioritize it accordingly.

**Comment by TorstenStueber** [comment]
@pendulum-chain/product even though this ticket is blocked by another ticket, shall we not move it to Ready, as it is ready?

**Comment by prayagd** [comment]
@TorstenStueber waiting for your and @Sharqiewicz estimates
